mysite/templates/mysite/project_bid_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "mysite/base_generic.html" %}
+
+{% block content %}
+
+<form method="post">{% csrf_token %}
+    Are you sure you want to delete bid on "{{ object.project.name }}" ?
+    <input type="submit" value="Submit" />
+</form>
+
+{% endblock %}
```

Comment:
why are you using a separate template for bid deletion confirmation? You should use a pop up confirmation OR javascript default confirmation box. On user's confirmation you can send a ajax request for delete action and update UI on success.

Reply:
Feedback Incorporated.